### PR TITLE
[W-15751811] Doc updates to client management pages

### PR DESCRIPTION
**Pull request by petery789**
# Notes 
Updates to OIDC client management page: 
* Move examples for PUT/DELETE requests out of the main configuration walkthrough into a separate section. 

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/configure-client-management-openid-task.adoc [hunk 1]
```diff
@@ -19,6 +19,7 @@ The following table contains examples of the URLs you need to supply, depending
 | Token Introspection | {BASE URL}/introspect | {BASE URL}/introspect | {BASE URL}/introspect
 |===
 
+== How to Configure
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-openid-task.adoc]:
```suggestion
== Configuring Client Management

```

File: modules/ROOT/pages/configure-client-management-openid-task.adoc [hunk 2]
```diff
@@ -65,51 +85,38 @@ Authorization: Bearer access-token
...
+
 +
 [NOTE]
 `client_name` in the request maps to `name` on the inbound side. +
 The authorization method depends on the grant type. For example, if the grant type is `implicit`, the `id_token` and `token` are returned.
-+
-This is an example of the DELETE request header:
-+
+
+This is another example of the DELETE request header:
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-openid-task.adoc]:
```suggestion
The following example shows another DELETE request header:
```

File: modules/ROOT/pages/configure-client-management-openid-task.adoc [hunk 3]
```diff
@@ -36,18 +37,37 @@ The *Add OIDC client provider* page appears.
...
+
+[NOTE]
+For Okta, the Okta admin needs to assign the dynamically generated clients to a user or a group of users in order for them to receive access tokens by sending over the Client ID and Client Secret.
+
+[NOTE]
+When Anypoint Platform makes an HTTP call to the configured OIDC endpoints, the HTTP call times out if no response after 5 seconds.
+
+== Client Application Updates and Deletes
+
+Anypoint platform embraces OpenID Connect Dynamic Client Registration, and hence client modification or deletion is not supported, although OKTA or another IDP might do so independently. But, as mentioned above we instead provide and support hooks for someone to implement API calls that are particular to their IDP. Below are samples for PUT and DELETE requests Anypoint will make.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-openid-task.adoc]:
```suggestion
Anypoint Platform embraces OpenID Connect Dynamic Client Registration. As a result, client modification and deletion are not supported, although OKTA or another IDP may allow it independently. Instead, Anypoint Platform provides and supports hooks so you can implement API calls that are particular to your IDP. The following examples show PUT and DELETE requests that Anypoint Platform will make.
```

File: modules/ROOT/pages/configure-client-management-openid-task.adoc [hunk 4]
```diff
@@ -36,18 +37,37 @@ The *Add OIDC client provider* page appears.
...
+* *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`. The `clientRegistrationUrl` is the value you configured in *Client Registration URL*. For example: `+https://identity.example.com/oauth2/connect/register/6779ef20e75817b79602+`. For detailed example, please see below.
+
+Once this is successfully configured, you can apply the OpenID Connect OAuth Token Enforcement policy to your API Gateways through API Manager. Requesting API access through API portals dynamically generates client applications in the configured IDP that acts as a token provider.
+
+[NOTE]
+For Okta, the Okta admin needs to assign the dynamically generated clients to a user or a group of users in order for them to receive access tokens by sending over the Client ID and Client Secret.
+
+[NOTE]
+When Anypoint Platform makes an HTTP call to the configured OIDC endpoints, the HTTP call times out if no response after 5 seconds.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-openid-task.adoc]:
```suggestion
Anypoint Platform makes an HTTP call to the configured OIDC endpoints, and if there is no response after five seconds, the HTTP call times out.
```

File: modules/ROOT/pages/configure-client-management-openid-task.adoc [hunk 5]
```diff
@@ -36,18 +37,37 @@ The *Add OIDC client provider* page appears.
...
+* *Enable client deletion in Anypoint Platform*: Enables deletion of clients created with this integration.
+* *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`. The `clientRegistrationUrl` is the value you configured in *Client Registration URL*. For example: `+https://identity.example.com/oauth2/connect/register/6779ef20e75817b79602+`. For detailed example, please see below.
+
+Once this is successfully configured, you can apply the OpenID Connect OAuth Token Enforcement policy to your API Gateways through API Manager. Requesting API access through API portals dynamically generates client applications in the configured IDP that acts as a token provider.
+
+[NOTE]
+For Okta, the Okta admin needs to assign the dynamically generated clients to a user or a group of users in order for them to receive access tokens by sending over the Client ID and Client Secret.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-openid-task.adoc:64]: I wouldn't put two notes in a row - is this something that can be moved higher in the article?

Comment (sarathecoubian) [reply modules/ROOT/pages/configure-client-management-openid-task.adoc:64]: Or combined somewhere

Comment (petery789) [reply modules/ROOT/pages/configure-client-management-openid-task.adoc:64]: Good question.. I just copied the previous docs and they were together. Let me try moving one.

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**